cli/provision.go

```diff
@@ -275,3 +298,47 @@ func channelsFromFile(path string) ([]mfxsdk.Channel, error) {
 
 	return channels, nil
 }
+
+func connectionsFromFile(path string) (mfxsdk.ConnectionIDs, error) {
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		return mfxsdk.ConnectionIDs{}, err
+	}
+
+	file, err := os.OpenFile(path, os.O_RDONLY, os.ModePerm)
+	if err != nil {
+		return mfxsdk.ConnectionIDs{}, err
+	}
+	defer file.Close()
+
+	connections := mfxsdk.ConnectionIDs{}
+	switch filepath.Ext(path) {
+	case ".csv":
```

Comment:
Maybe move this to constants or no need?

Reply:
IMO no need if it only appears here

Reply:
I believe this is used in 3 places.  I was thinking I had switched the others to constants previously.  I will switch this and the json extension over